plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestKuduConnectorTest.java

```diff
@@ -951,6 +951,38 @@ public void testUpdate()
     @Override
     public void testUpdateMultipleCondition() {}
 
+    /**
+     * This test fails intermittently because Kudu doesn't have strong enough
+     * semantics to support writing from multiple threads.
+     */
+    @Test
+    @Disabled
```

Comment:
The following test body is redundant if we disable the test.

Reply:
Do you want me to remove

Reply:
Ideally, we should run this test by isolating other operations with `executeExclusively`.